core/include/scipp/core/eigen.h

```diff
@@ -4,6 +4,7 @@
 /// @author Simon Heybrock
 #pragma once
 #include <Eigen/Core>
+#include <Eigen/Geometry>
```

Comment:
Why?

Reply:
Linker error otherwise of the form:
```
undefined reference to `Eigen::Matrix<double, 1, 3, 2, 1, 3> Eigen::MatrixBase<Eigen::Block<Eigen::Matrix<double, 3, 3, 2, 3, 3>, 1, 3, 1, 32> >::cross<Eigen::Block<Eigen::Matrix<double, 3, 3, 2, 3, 3>, 1, 3, 1, 32> >(Eigen::MatrixBase<Eigen::Block<Eigen::Matrix<double, 3, 3, 2, 3, 3>, 1, 3, 1, 32> > const&) const'
```

Reply:
Would it be enough to include this where the function is implemented, rather than everywhere in scipp?

Reply:
That should work.